action_page_user.php

```diff
@@ -8,49 +8,39 @@
 
 	// check to see if all user-input fields are filled in
 	if ( ($_POST["firstName"]==null) or ($_POST["lastName"]==null) or ($_POST["email"]==null)){
-		$inputerror = true;
-		header('Refresh: 2; url=input_user.html');
+		header('Location: input_user.html');
 		die ("ERROR: Please fill in all required fields.");
 	}
+		
+	// get credentials from user.credentials.ini
+	$config = parse_ini_file('user.credentials.ini');
```

Comment:
Interesting way of doing it. Most times I see just a php file and someone doing a require_once against it. This works though